src/data/move.ts

```diff
@@ -7924,7 +7924,15 @@ export function initMoves() {
     new StatusMove(Moves.REFLECT_TYPE, Type.NORMAL, -1, 15, -1, 0, 5)
       .attr(CopyTypeAttr),
     new AttackMove(Moves.RETALIATE, Type.NORMAL, MoveCategory.PHYSICAL, 70, 100, 5, -1, 0, 5)
-      .partial(),
+      .attr(MovePowerMultiplierAttr, (user, target, move) => {
+        const turn = user.scene.currentBattle.turn;
+        const lastPlayerFaint = user.scene.currentBattle.playerFaintsHistory[user.scene.currentBattle.playerFaintsHistory.length -1];
+        const lastEnemyFaint = user.scene.currentBattle.enemyFaintsHistory[user.scene.currentBattle.enemyFaintsHistory.length -1];
+        return (
+          lastPlayerFaint !== undefined && turn - lastPlayerFaint.turn === 1 && user.isPlayer() ||
+          lastEnemyFaint !== undefined && turn - lastEnemyFaint.turn === 1 && !user.isPlayer()
```

Comment:
```suggestion
          (lastPlayerFaint !== undefined && turn - lastPlayerFaint.turn === 1 && user.isPlayer()) ||
          (lastEnemyFaint !== undefined && turn - lastEnemyFaint.turn === 1 && !user.isPlayer())
```
Actually this should be changed for clarity too